README.md

```diff
@@ -1,8 +1,19 @@
-# ProtonVPN-CLI
+<h1 align="center">ProtonVPN-CLI</h1>
+<p align="center">
+  <img src="https://i.imgur.com/tDrwkX5l.png" alt="Logo"></img>
+</p>
+
+<p align="center">
+    <img alt="Downloads" src="https://pepy.tech/badge/protonvpn-cli">
+    <img alt="GitHub" src="https://img.shields.io/github/license/ProtonVPN/protonvpn-cli-ng">
+    <img alt="Downloads per Week" src="https://pepy.tech/badge/protonvpn-cli/week">
+    <br>
+    <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/ProtonVPN?style=social">
+    <img alt="Subreddit subscribers" src="https://img.shields.io/reddit/subreddit-subscribers/ProtonVPN?label=Join%20r%2FProtonVPN&style=social">
+</p>
+
+<h3 align="center">A Linux CLI for ProtonVPN. Written in Python.</h3>
```

Comment:
Why are you replacing MarkDown with HTML?

Reply:
I think centering the top part looks a lot better. As it's just for the header I don't see that much of an issue. But generally I agree.